solutions/tensorflow_lite/label_image.py

```diff
@@ -0,0 +1,102 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""label_image for tflite."""
```

Comment:
Since these files are in both places solutions and samples, can it be removed from samples?

Reply:
I think it's valuable to keep the TensorFlow sample as it demonstrates how to debug as well as some limitation Mystikos currently has https://github.com/deislabs/mystikos/blob/main/samples/tensorflow_lite/Dockerfile.bionic#L1